docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -19,15 +19,17 @@ import { PlatformTags } from '~/ui/components/Tag';
 
 `expo-notifications` provides an API to fetch push notification tokens and to present, schedule, receive and respond to notifications.
 
-> **warning** Push notifications (remote notifications) functionality provided by `expo-notifications` is unavailable in Expo Go from SDK 53. A [development build](/develop/development-builds/introduction/) is required to use push notifications. Scheduled notifications (local notifications) remain available in Expo Go.
+This page is a reference document for the package. There are also comprehensive [guides available](/push-notifications/overview/).
```

Comment:
Let's use BoxLink here to redirect towards Push notification section. We don't have to specify that this is an API page since that is self-explanatory.

Reply:
done